networkx/algorithms/coloring/tests/test_coloring.py

```diff
@@ -429,6 +431,54 @@ def test_hardest_prob(self):
         )
         check_state(**params)
 
+    def test_strategy_saturation_largest_first(self):
+        def color_remaining_nodes(G, colored_vertices):
+            color_assignments = []
+            aux_colored_vertices = {
+                key: value for key, value in colored_vertices.items()
+            }
```

Comment:
Is this different than `colored_vertices.copy()`? If not, the copy version is more readable.

Reply:
Good point, I will amend to use copy()